Scripts/Levels/Level Main.gd

```diff
@@ -141,18 +160,29 @@ func process_actions(actions):
 
 var turn_end_actions = []
 var turn_start_actions = []
+# Reserved for moves that only activate for the player who casted them
+var player_turn_actions = []
 
-func register_repeated_action(object, method, num_times, siignal):
+# Format for actions
+# [Organism, Ability_Name, Duration, Target Player]
+func register_repeated_action(object, method, num_times, player, siignal):
```

Comment:
Make null a default argument for player. Remove all references player_turn_actions. Big shark guy can just use turn_start_actions with the player object argument.